### PR TITLE
Thanhson profile

### PR DESCRIPTION
**Pull request by Thanhson89**

### CODE REVIEW

File: _persons/thanhson.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+# use person if you want to have a personal website created
+layout: person
+first_name: Thanh Son
+last_name: Nguyen
+key: thanhson
```
Comment (zvonimir) [inline _persons/thanhson.md]: Your key should be your last name. So you should change that everywhere. Everything else looks good.

**Review by zvonimir — CHANGES_REQUESTED** [review]

File: _persons/thanhson.md (hunk 1 above)
Comment (Thanhson89) [reply _persons/thanhson.md]: I changed it

File: _persons/thanhson.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# use person if you want to have a personal website created
+layout: person
+first_name: Thanh Son
+last_name: Nguyen
+key: nguyen
+permalink: /team/nguyen/
+# faculty | phdstudent | phd_rotation | postdoc | msstudent | collaborator
+role: phdstudent
+email: thanhson@cs.utah.edu
+image: /assets/images/people/thanhson.jpg
```
Comment (zvonimir) [inline _persons/thanhson.md:11]: The size of your headshot image should be 400x400 pixels. Sorry, I did not notice that sooner.

**Review by zvonimir — CHANGES_REQUESTED** [review]
Looks good for the most part.

File: _persons/thanhson.md (hunk 2 above)
Comment (Thanhson89) [reply _persons/thanhson.md:11]: I updated the image

**Review by zvonimir — APPROVED** [review]
Looks good.

**Pull request merged**